app/res/values/strings.xml

```diff
@@ -352,4 +352,17 @@ Some images in CommCare are graciously used under Creative Commons Licensing fro
     <string name="sign_button" cc:translatable="true">Gather Signature</string>
     <string name="markup_image" cc:translatable="true">Markup Image</string>
     <string name="draw_image" cc:translatable="true">Sketch Image</string>
+
+    <!--Incorrect Target Screen-->
```

Comment:
Since none of these are being referenced from a layout file, I would have a slight preference for moving them to `android_translatable_strings.txt`. I think it's good for us to stick to some rule about which strings we put in which files.

Reply:
I put these in here since it results in better IDE/Gradle support in comparison to `android_translatable_strings.txt` . For ex. If one of these strings get deleted by mistake the build will break and we will get indication of something is wrong. I am of opinion that we should try to avoid `android_translatable_strings.txt`  for this reason unless it's absolutely necessary.

Reply:
Hadn't thought about it that way, that's fair.